### ISSUE TITLE
H5Class_overview currently not working

### ISSUE THREAD

**Issue by mannau** [hhoeflin/hdf5r#65]
`H5Class_overview` is currently not working since `inst/manual/function_overview.html` is missing:
```r
> H5Class_overview()
Error in browseURL(system.file("manual/function_overview.html", package = "hdf5r"),  : 
'url' must be a non-empty character string
```

We should either:
- Add the manual file `inst/manual/function_overview.html`
- Delete `H5Class_overview` and all references to it

**Comment by hhoeflin** [comment]
yes - I plan to eliminate that. I added the file to .Rbuildignore, not realizing that it would cause this bug.

The function overview is currently only partial anyway. I plan to try and provide a one-page version of the complete manual in html instead. May be more useful for searching. Otherwise the pdf-manual may be ok as well.

**Comment by hhoeflin** [comment]
Will remove H5Class_overview from everywhere in the documentation tomorrow.

**Comment by hhoeflin** [comment]
Fixed in PR #67
